From 96819c0d6bdea68c194d828d7f2bd27920fdb480 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 16 Oct 2018 11:17:36 +0800
Subject: [PATCH 4918/5242] MLK-19947-2 soc: imx: select CLKSRC_IMX_GPT for
 i.MX8QXP

commit  fce27d5ce1d44390532ab6c3ae76580c58e7e4c3 from
https://source.codeaurora.org/external/imx/linux-imx.git

Select CLKSRC_IMX_GPT for i.MX8QXP by default.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Acked-by: Leonard Crestez <leonard.crestez@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/soc/imx/Kconfig |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/imx/Kconfig b/drivers/soc/imx/Kconfig
index 7c6897a..3a1fa18 100644
--- a/drivers/soc/imx/Kconfig
+++ b/drivers/soc/imx/Kconfig
@@ -19,6 +19,7 @@ config ARCH_FSL_IMX8QXP
 	select HAVE_IMX_MU
 	select HAVE_IMX_RPMSG
 	select HAVE_IMX8_SOC
+	select CLKSRC_IMX_GPT
 
 config ARCH_FSL_IMX8MQ
 	bool "i.MX8MQ"
-- 
1.7.9.5

